timescaledb/tutorials/grafana/visualizations/time-series.md

```diff
@@ -0,0 +1,244 @@
+    to get familiar with creating your first dashboard and visualizations before
+    you start.
+</highlight>
+
+
+The examples in this section use these variables and Grafana functions:
+* `$symbol`: a variable used to filter results by stock symbols.
+* `$bucket_interval`: the interval size to pass to the `time_bucket`
+  function when aggregating data.
+* `$__timeFrom()::timestamptz` & `$__timeTo()::timestamptz`:
+  Grafana variables. You change the values of these variables by
+  using the dashboard's date chooser when viewing your graph.
+
+
+## Create a time-series graph with raw data
+
+A very common use case of the time-series graph is displaying stock data. The
+graph makes it easy to see if the value of a stock is going up or down. Also, no
+extra calculations are needed to make the graph.
+
+<procedure>
+
+### Creating a time-series graph with raw data
+
+1.  Add the `$symbol` variable of type `Text box` to the Grafana dashboard.
+
+1.  In Grafana, create a new panel and add this query:
+
+    ```SQL
+    SELECT time,
+        price
+    FROM stocks_real_time
+    WHERE symbol = '$symbol'
+        AND time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+    ORDER BY time;
+    ```
+
+1.  Enter `AMD` in the `symbol` variable. Adjust the time range of your 
+    dashboard if desired. This plot uses a raw query, so if you set a large
+    time range, it might take a long time for Grafana to get all the rows and
+    display the data.
+
+1.  When 'Table view' is selected, the returned data looks like this:
+
+    ```bash
+    time                | price |
+    --------------------+--------+
+    2022-02-08 06:38:21 |  123  |
+    2022-02-08 06:38:28 |  123  |
+    2022-02-08 06:39:18 |  123  |
+    2022-02-08 06:39:56 |  123  |
+                …       |   …   |
+    ```
+
+    Check that your data meets Grafana's requirements for graphing time series.
+    The data must have a column named `time`, containing timestamps. Other
+    columns can be named as you like. For the time-series visualization, the
+    timestamps must be in ascending order. Otherwise, you get an error.
+
+1.  Select `Time series` as your visualization type.
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/time-series/time-series-visualization-type.png" alt="Screenshot of the Grafana dashboard. The 'Visualizations' tab is focused. Underneath, 'Time-series' shows as a visualization type."/>
+
+1.  Grafana returns a graph that looks similar to this:
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/time-series/simple-time-series-graph.png" alt="Screenshot of the time-series graph produced by Grafana. The graph represents the price of AMD in the past 6 hours."/>
+
+</procedure>
+
+## Create a time-series graph from pre-aggregated data using Continuous Aggregates
```

Comment:
Ah, now I understand why I was confused about in my thinking during our recent conversations.

You aren't showing continuous aggregates at all here. This is just an aggregate query using Time_bucket().

So, change heading here in kind.